cmd/vicadmin/usersession.go

```diff
@@ -112,6 +119,13 @@ func (u *UserSessionStore) reaper() {
 	}
 }
 
+// Destroy will logout and delete all sessions in the store, irrespective of expiration
+func (u *UserSessionStore) Destroy() {
+	for id := range u.sessions {
```

Comment:
I would go a different route. I would grab a lock, logout from all sessions, and then wipe out u.sessions dictionary. the problem with iterating over a live sessions dictionary can lead to concurrency problems in general. Locking full access to u.sessions while Destroy is running would make it a lot safer than locking access for each element.